### ISSUE TITLE
Nest circles to indicate subcircle

### ISSUE THREAD

**Issue by leilatite** [gather-community/gather#919]
Is it possible to nest subcircles underneath a main circle. See [issue #917](https://github.com/gather-community/gather/issues/917)

**Comment by Ruthie-100** [comment]
I agree having the ability to nest sub circles under a main circle would be fantastic!
Meanwhile, I'd appreciate adding subcircles, helping circles and operating roles to the list of type choices for groups. Just having those names on the list would be a big help. Thanks!
